embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/logs/EmbraceExtensions.kt

```diff
@@ -74,9 +75,8 @@ internal fun LogRecordBuilder.setSessionId(sessionId: String): LogRecordBuilder
 /**
  * Set an id for the log
  */
-internal fun LogRecordBuilder.setEventId(eventId: String): LogRecordBuilder {
+internal fun ReadWriteLogRecord.setEventId(eventId: String) {
```

Comment:
Is this the same as `log_id`?

Reply:
Yes, it should actually be `log_id` instead of `event_id`, I'll change it.